clang/test/CIR/Lowering/address-space.cir

```diff
@@ -3,7 +3,10 @@
 
 !s32i = !cir.int<s, 32>
 
-module  {
+module attributes {
+  cir.triple = "spirv64-unknown-unknown",
+  llvm.data_layout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024-G1"
```

Comment:
Not for this PR, but opening for discussion: we should make `llvm.data_layout` computable from `cir.triple`. @sitio-couto do we already have that or plans to do so?